README.md

```diff
@@ -450,6 +450,38 @@ await build({
 
 1. Ensure the workflow will run on tag creation. For example, see [Trigger GitHub Action Only on New Tags](https://stackoverflow.com/q/61891328/188246)).
 
+### Using Another Package Manager
+
+For some reasons you may want to use another Node.js package manager, such as Yarn or pnpm.
+You can override the `packageManager` option in build options. Default value is `npm`.
+
+If you're using Yarn:
+
+```ts
+await build({
+  // ...etc...
+  packageManager: "yarn",
+});
+```
+
+If you're using pnpm:
+
+```ts
+await build({
+  // ...etc...
+  packageManager: "pnpm",
+});
+```
```

Comment:
Maybe to help shorten this section we could remove this example and just do the following above:

```ts
packageManager: "yarn", // or "pnpm"
```